Dockerfile

```diff
@@ -6,4 +6,4 @@ RUN pecl install swoole \
 RUN curl -sS https://getcomposer.org/installer | php && \
     mv composer.phar /usr/local/bin/composer
 
-RUN apt-get update && apt-get install -y zip
+RUN apt-get update && apt-get install -y zip sqlite3
```

Comment:
Not seeing the php-sqlite3 package here does that mean it's already included in the image and just this package was missing?

Reply:
Only this package was missing.

Reply:
:+1